TesseractOCR/G8Tesseract.mm

```diff
@@ -93,16 +105,25 @@ - (id)initPrivateWithDataPath:(NSString *)dataPath
         _monitor->cancel = (CANCEL_FUNC)[self methodForSelector:@selector(tesseractCancelCallbackFunction:)];
         _monitor->cancel_this = (__bridge void*)self;
 
-        if (dataPath != nil) {
-            [self copyDataToDocumentsDirectory];
+        if (_absoluteDataPath != nil) {
+            // config Tesseract to search trainedData in tessdata folder of the Caches folder
+            NSArray *cachesPaths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
+            NSString *cachesPath = cachesPaths.firstObject;
+
+            _absoluteDataPath = [cachesPath stringByAppendingPathComponent:_absoluteDataPath].copy;
+
+            [self moveTessdataToCachesDirectoryIfNecessary];
         }
         else {
-            [self setUpTesseractToSearchTrainedDataInTrainedDataFolderOfTheApplicatinBundle];
+            // config Tesseract to search trainedData in tessdata folder of the application bundle];
+            _absoluteDataPath = [NSString stringWithFormat:@"%@/", [NSString stringWithString:[NSBundle bundleForClass:self.class].bundlePath]].copy;
         }
+        
+        setenv("TESSDATA_PREFIX", [_absoluteDataPath stringByAppendingString:@"/"].UTF8String, 1);
```

Comment:
Double // is a correct path, as I know. Anyway, I'll remove / from the else block, so there will be only one /.